src/components/pages/profile/account/SetMFA.vue

```diff
@@ -403,7 +427,7 @@ const disableMfaAndNotify = async () => {
 /** Called when an error occurs while creating MFA */
 const onCreateError = (error: unknown) => {
   if (
-    isRequestError<ContactMfaCreateApiErrorData>(error, undefined, 401) &&
+    isRequestError<ContactMfaCreateApiErrorData>(error, undefined, [401]) &&
     error.response.data.message === LOGIN_CODES.INVALID_TOKEN
```

Comment:
Why is this checking `data.message` but everywhere else it checks `data.code`?

Reply:
Yes, there are still legacy issues from Passport, but I also set `code` in the backend, so this could already work with `code`